acceptance/acceptance_test.go

```diff
@@ -353,8 +369,10 @@ func CopyDir(src, dst string, inputs, outputs map[string]bool) error {
 			return err
 		}
 
-		if strings.HasPrefix(name, "out") {
-			outputs[relPath] = true
+		if strings.HasPrefix(relPath, "out") {
```

Comment:
Is this safe? 

Before: `foo/bar/out.txt` would be classified as an output. 

Now, it would be classified as input. There's a risk that any acceptance tests that use that pattern will start silently passing without asserting on the contents of `foo/bar/out.txt`